node-tests/blueprints/controller-test.js

```diff
@@ -13,6 +13,7 @@ const expect = chai.expect;
 
 const generateFakePackageManifest = require('../helpers/generate-fake-package-manifest');
 const enableModuleUnification = require('../helpers/module-unification').enableModuleUnification;
+const enableOctane = require('../helpers/setup-test-environment').enableOctane;
```

Comment:
I recommend to include the `enableModuleUnification ` function into the `/helpers/setup-test-environment` module, but I would like to do that in an independent PR.